simulationdataschema/model_system.py

```diff
@@ -761,37 +740,44 @@ def normalize(self, archive, logger):
 
 
 class ModelSystem(System):
+    # ! Work in this description with new `AtomState`
     """
-    Model system used as an input for the computation. It inherits from `System` where a set
-    of sub-sections for the `elemental_composition` is defined.
+    Model system used as an input for the computation.
 
-    We also defined:
+    We define:
         - `name` refers to all the verbose and user-dependent naming in ModelSystem,
         - `type` refers to the type of the ModelSystem (atom, bulk, surface, etc.),
-        - `dimensionality` refers to the dimensionality of the ModelSystem (0D, 1D, 2D, 3D),
+        - `dimensionality` refers to the dimensionality of the ModelSystem (0, 1, 2, 3),
 
-    If the ModelSystem `is_representative`, the normalization occurs. The time evolution of
-    the system is encoded on the fact that ModelSystem is a list under Simulation, and for
-    each element of that list, `time_step` can be defined.
+    If the ModelSystem `is_representative`, normalization occurs. The time evolution of the
+    ModelSystem is encoded on the fact that it is a list under `Simulation`, and for each element
+    of that list, `time_step` can be defined.
```

Comment:
> If the ModelSystem `is_representative`, normalization occurs.

If `ModelSystem` `is_representative`, proceed with normalization.

> The time evolution of the ModelSystem is encoded on the fact that it is a list under `Simulation`

The time evolution of `ModelSystem` is stored in a `list` format under `Simulation`.